packages/Webkul/Installer/src/Http/Controllers/InstallerController.php

```diff
@@ -168,7 +170,9 @@ public function adminConfigSetup()
 
             return true;
         } catch (\Throwable $th) {
-            dd($th);
+            Log::error('Error in Admin installer config setup'.$th->getMessage());
```

Comment:
Add translation here for all local.

Reply:
> Add translation here for all local.

I have searched with `\Log::error('` there is a used text message. can you please guide me?
Thanks in advance.

Reply:
> > Add translation here for all local.
> 
> I have searched with `\Log::error('` there is a used text message. can you please guide me? Thanks in advance.

You can use the trans method to write the translation and create the new key inside the app.php file for all local.

For Example: `\Log::error(trans('installer::app.install-error'))`